sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaIOTest.java

```diff
@@ -731,7 +833,24 @@ public Long deserialize(String topic, byte[] bytes) {
 
     @Override
     public void close() {
+    }
+  }
+
+  // class for testing coder inference
+  private static class ObjectDeserializer
+          implements Deserializer<Object> {
```

Comment:
Maybe put something else other than Object, cause I'd think that Object can be inferred by mistake